src/app/page/Signup/SignupScene.jsx

```diff
@@ -97,8 +97,7 @@ function SignupScene(props) {
         payload = { ...payload, location: location };
       }
     } catch (error) {
-      console.log("ERROR WHEN GETTING LOCATION", error);
-      console.log("address: ", location.address);
+      setErrorSnackbarMessage(error);
```

Comment:
Would it be okay if you have another commit that removes this old implementation of snackbar, and use the new snackbar using the callback provided in the snackbar context? cleaner/newer pattern and this should be throwing deprecation warnings in the console

Reply:
Definitely! I’ll do that tonight. I’ll also do that to the other snack bar in the component when updating the user.